[shopsys] remove deprecations from 7.x.x (#1193)
shopsys/shopsys: docs/upgrade/UPGRADE-unreleased.md

    - check whether all deprecated methods from multiple cron commands implementation are replaced with the new ones based on ([#817](https://github.com/shopsys/shopsys/pull/817))
    - check whether all deprecated methods and configurations for redis clients and redis commands are removed and modified to use new way based on ([#1161](https://github.com/shopsys/shopsys/pull/1161)) and ([#886](https://github.com/shopsys/shopsys/pull/886))
        - remove registration of `Shopsys\FrameworkBundle\Command\RedisCleanCacheOldCommand: ~` from `commands.yml`
    - check whether you extended classes that constructor signature was changed and fix them
        - `Shopsys\FrameworkBundle\Model\Mail\MailTemplateFacade`
        - `Shopsys\FrameworkBundle\Component\Router\DomainRouterFactory`
        - `Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExporter`
        - `Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportStructureFacade`
        - `Shopsys\FrameworkBundle\Model\Product\ProductFacade`
        - `Shopsys\FrameworkBundle\Model\Product\ProductVariantFacade`
        - `Shopsys\FrameworkBundle\Component\Elasticsearch\ElasticsearchStructureManager`

>     - check wether you have overwritten methods `createQueryBuilder` and renamed method `getProductTotalCountForDomainAndLocale` that is now named `getProductTotalCountForDomain` of `Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportWithFilterRepository` class and fix their signature

Reviewer: ```suggestion
    - check wether you have overwritten methods of `Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportWithFilterRepository` class and fix their signatures
        - `createQueryBuilder` has less parameters
        - `getProductTotalCountForDomainAndLocale` was renamed to `getProductTotalCountForDomain` and has less parameters
```
